content/blog/2020/08/2020-08-26-machine-learning-plugin-coding-phase3.adoc

```diff
@@ -0,0 +1,86 @@
+---
+layout: post
+title: "Machine Learning Plugin project - Coding Phase 2 blog post"
+tags:
+- machinelearning
+- datascience
+- plugins
+- communitybonding
+- gsoc
+- gsoc2020
+author: loghijiaha
+description: "Google Summer of Code 2020 - Machine Learning Plugin project update for Coding Phase 2"
+opengraph:
+  image: /images/gsoc/2020/machine_learning_plugin/jenkins-gsoc-machine-learning-opengraph.png
+---
+
+image::/images/gsoc/jenkins-gsoc-logo_small.png[align="center"]
+
+Good to see you all again !
+
+This is my final blog post about coding phase 3 in Jenkins link:/projects/gsoc/2020/projects/machine-learning/[Machine Learning Plugin] for GSoC 2020.
+Being at the end of GSoC 2020, we had to finish all the implementation works and testing before a stable release in the main repository. Throughout this program, there were lots of learning and hard work will make this plugin valuable to the Data Science and Jenkins community.
+
+== Summary
+
+With combining all of the work in phase 1, 2 and 3, initial version of Machine learning plugin( 1.0 ) was successfully released in Jenkins plugin repository.
+An interesting feature which allows users to connect to their existing language kernels more than connecting to only IPython kernel was introduced in this phase. It can be selected in multiple steps with different kernel.
+Images and graphs produced by Jupyter notebooks will be saved in user preferred folder in the workspace that can be used for reporting/analytic purposes later. Hoping this blog concises about Machine Learning Plugin's features and future contributions. Thank you for your interest and support !!!
```

Comment:
I think concise can be used only as adjective. So maybe "Hoping this blog summarizes the Machine Learning ..."?